backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/icon/BpkIcon.kt

```diff
@@ -51,6 +52,7 @@ class BpkIcon internal constructor(
 
 enum class BpkIconSize {
     Small,
+    Medium,
```

Comment:
the icon is the large size (you always need to select the parent of the icon to ensure you get the icon size, rather than just that specific svg size without any border) - no need to add a new icon size

Reply:
Sorry, I don't understand what you're saying here

Reply:
I got it, on the Figma